### DISCUSSION TITLE
Get currrent logged user with custom data added to the User Collection type

### DISCUSSION THREAD

**Discussion by Antoine-lb** [strapi/strapi#8601] [Legacy Topics]
Hello 👋

I'm using `users/me` endpoint to fetch all the data of the current user (learn about that in [this dicussion](https://github.com/strapi/strapi/discussions/7453)).

The problem is that I added data to the `User` collection type, but that custom field is not being fetched by the `GET /users/me` end point.

More details of my problem:
Basically I created a `many to many` relationship between users and restaurants. Not all users have access to all restaurants (they are administrators). So I added a field to act like a role, now the user is allowed to edit all of the restaurants that are linked with it.

I would like to fetch that data with the `/users/me` end point, but is only returning the default data, and not my new `restaurant` field.

Am I doing it right? Do I have to make two requests to get the data about the current user?

Thanks for the help

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by Antoine-lb** [reply]
Created the same question: https://forum.strapi.io/t/get-currrent-logged-user-with-custom-data-added-to-the-user-collection-type/1024

**Comment by Antoine-lb** [comment]
Question answered in: https://forum.strapi.io/t/get-currrent-logged-user-with-custom-data-added-to-the-user-collection-type/1024
